CONTRIBUTING.md

```diff
@@ -10,7 +10,31 @@ We actively welcome your pull requests.
 3. If you've changed APIs, update the documentation.
 4. Ensure the test suite passes.
 5. Make sure your code lints.
-6. If you haven't already, complete the Contributor License Agreement ("CLA").
+6. Run `black` formatting.
```

Comment:
do you use any settings or you already have a configuration in the repo for `black`?

Reply:
`black` is already part of our `lint_python` workflow. we use default settings